modules/python/compat/compat-python.h

```diff
@@ -31,6 +31,11 @@
 #if (SYSLOG_NG_ENABLE_PYTHONv2)
 #define PYTHON_BUILTIN_MODULE_NAME "__builtin__"
 #define PYTHON_MODULE_VERSION "python2"
+
+/*
+ * Macro from python3 Include/datetime.h file
+ */
+#define PyDateTime_DELTA_GET_SECONDS(o)      (((PyDateTime_Delta*)o)->seconds)
```

Comment:
Can you add an `#ifndef` guard around it?

Reply:
What do you want to achieve with that ?

Reply:
I was thinking, if at some point someone wants to include this python3/timestamp.h header, they would run into trouble. It also occurred me that you should have included it instead of picking a macro. But this macro is not available in python2, so you would still have to deal with compatibility, so we wouldn't have won anything. Just picking this line is sufficient.

I do not want to force this, just me seems a good practice to add guard in such cases.

Reply:
This is a compatibility layer, and the macro exists only in python3, so I only provided it for python2 (anybody includes python3 won't get in trouble).

So one should always include `datetime.h` if want this macro, and as well the compat (maybe it would worth to make the same structure as for other compat headers in syslog-ng).

If this macro and the one in python3 is colliding somebody screw up big time, and I want to put the pain on them instead of silently hiding the issue.

Reply:
Okok, I agree.